common/controlsigs.h

```diff
@@ -22,6 +22,7 @@
 #define uiSeparatorSignature 0x53657061
 #define uiSliderSignature 0x536C6964
 #define uiSpinboxSignature 0x5370696E
+#define uiSpinboxDoubleSignature 0x5370696E
```

Comment:
```suggestion
#define uiSpinboxDoubleSignature 0x53706E44
```
If you add a new control, each control type needs to be assigned a unique type ID value.
(The type IDs are actually ASCII to assist in debugging: `0x5370696E` = "Spin", `0x53706E44` = "SpnD".)